### PR TITLE
GitHub integration UI

### PR DESCRIPTION
**Pull request by SvetaMemphis**

### CODE REVIEW

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+                                                    <img src={tickCircle} className="connected" alt="connected" />
+                                                    &nbsp;Connected
+                                                </span>
+                                            }
+                                            colorType="white"
+                                            radiusType="circle"
+                                            backgroundColorType="green"
+                                            fontSize="12px"
+                                            fontFamily="InterSemiBold"
+                                            // isLoading={loadingSubmit}
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:230]: remove comment

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+                                        <Button
+                                            width="70px"
+                                            height="22px"
+                                            placeholder={'Connect'}
+                                            colorType="white"
+                                            radiusType="circle"
+                                            backgroundColorType="purple"
+                                            fontSize="12px"
+                                            fontFamily="InterSemiBold"
+                                            // isLoading={loadingSubmit}
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:244]: When pressing connect, you can activate the loader inside the button

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+                                            // disabled={isValue && !creationForm.isFieldsTouched()}
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:245]: Please disable the connect button until the secret key is inserted

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/style.scss [hunk 4]
```diff
@@ -0,0 +1 @@
+//
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/components/gitHubIntegration/style.scss:1]: delete the file

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+                                width="140px"
+                                height="35px"
+                                placeholder="Integration guide"
+                                colorType="white"
+                                radiusType="circle"
+                                backgroundColorType="purple"
+                                border="none"
+                                fontSize="12px"
+                                fontFamily="InterSemiBold"
+                                onClick={() => window.open('https://docs.memphis.dev/memphis/dashboard-ui/integrations/notifications/slack', '_blank')}
```
Comment (avrhamNeeman) [inline ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:204]: You redirect to Slack.

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/style.scss (hunk 4 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/administration/integrations/components/gitHubIntegration/style.scss:1]: done

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+                                        onBlur={(e) => updateKeysState('token', e.target.value)}
+                                        onChange={(e) => updateKeysState('token', e.target.value)}
+                                        value={formFields?.keys?.token}
+                                    />
+                                </Form.Item>
+                            </div>
+                            {isValue && (
+                                <div className="input-field">
+                                    <p className="title">Repos</p>
+                                    {/* <span className="desc">Lorem Ipsum is simply dummy text of the printing and typesetting industry.</span> */}
```
[the remark this answers is not in the document]
Comment (SvetaMemphis) [reply ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:280]: Done

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js (hunk 3 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:245]: Done

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js (hunk 2 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:244]: Done

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js (hunk 1 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:230]: Done

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+        }
+    });
+    const [loadingSubmit, setLoadingSubmit] = useState(false);
+    const [loadingDisconnect, setLoadingDisconnect] = useState(false);
+    const [imagesLoaded, setImagesLoaded] = useState(false);
+    const [repos, setRepos] = useState([]);
+    const [addNew, setAddNew] = useState(false);
+
+    useEffect(() => {
+        console.log(value);
```
[the remark this answers is not in the document]
Comment (SvetaMemphis) [reply ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:47]: Done

File: ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js (hunk 5 above)
Comment (SvetaMemphis) [reply ui_src/src/domain/administration/integrations/components/gitHubIntegration/index.js:204]: put in comments until ill have the url

File: ui_src/src/const/integrationList.js [hunk 8]
```diff
@@ -270,6 +272,34 @@ export const INTEGRATION_LIST = {
...
     },
+    GitHub: {
+        name: 'Github',
+        by: 'memphis',
+        banner: <img className="banner" src={githubBanner} alt="gitHubBanner" />,
+        insideBanner: <img className="insideBanner" src={githubBannerPopup} alt="slackBannerPopup" />,
+        icon: <img src={githubIntegrationIcon} alt="gitHubIcon" />,
+        description:
+            'GitHub is an open source code repository and collaborative software development platform. Use GitHub repositories to manage your Schemaverse schemas and Functions source code.',
+        category: CATEGORY_LIST['All'],
```
[the remark this answers is not in the document]
Comment (SvetaMemphis) [reply ui_src/src/const/integrationList.js]: Done

**Pull request merged**